src/iOS/Perspex.iOS/iOSPlatform.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Reflection;
+using Perspex.Input;
+using Perspex.Input.Platform;
+using Perspex.iOS;
+using Perspex.Platform;
+using Perspex.Shared.PlatformSupport;
+using Perspex.Skia;
+using UIKit;
+
+namespace Perspex
+{
+    public static class iOSApplicationExtensions
+    {
+        public static AppT UseiOS<AppT>(this AppT app) where AppT : Application
+        {
+            Perspex.iOS.iOSPlatform.Initialize();
+            return app;
+        }
+
+        // I wish I could merge this with the SkiaPlatform itself. Might be possible
+        // once we switch to SkiaSharp
```

Comment:
Yah this is from my other PR (for the app configure cleanup) that was dependent on the Skia port. Will adjust this.